blazor/diagram/tool-tip.md

```diff
@@ -19,4 +19,656 @@ By default, the diagram displays a tooltip that provides information about the s
+    private void ContentChange()
+    {
+        connector[0].Tooltip.Content = "UpdateTooltipContent";
+    }
+}
+```
+
+## How to shows/hides the tip pointer for tooltip
+
+[ShowTipPointer](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_ShowTipPointer) property value is true, if the tip pointer is visible; otherwise, false.
+
+The following code example is used to set tooltip tip pointer for nodes.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Node TipPointer" @onclick="TipPointerChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Nodes="@nodes" />
+@code
+{
+    DiagramObjectCollection<Node> nodes;
+
+    protected override void OnInitialized()
+    {
+        nodes = new DiagramObjectCollection<Node>();
+        Node node = new Node()
+        {
+            ID = "node1",
+            OffsetX = 250,
+            OffsetY = 250,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="NodeTooltip",ShowTipPointer=true},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        nodes.Add(node);
+    }
+    //Change TipPointer at run time
+    private void TipPointerChange()
+    {
+        nodes[0].Tooltip.ShowTipPointer = false;
+    }
+}
+```
+The following code example is used to set tooltip tip pointer for connectors.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Connector TipPointer" @onclick="TipPointerChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Connectors="connector" />
+@code
+{
+    //Defines diagram's connector collection.
+    DiagramObjectCollection<Connector> connector;
+
+    protected override void OnInitialized()
+    {
+        connector = new DiagramObjectCollection<Connector>();
+        Connector connectors = new Connector()
+        {
+            ID = "Connector1",
+            SourcePoint = new DiagramPoint() { X = 500, Y = 500 },
+            TargetPoint = new DiagramPoint() { X = 600, Y = 400 },
+            Tooltip = new DiagramTooltip(){Content="ConnectorTooltip",ShowTipPointer=true},
+            Constraints =  ConnectorConstraints.Default|ConnectorConstraints.Tooltip ,
+        };
+        connector.Add(connectors);
+    }
+    //Change TipPointer at run time
+    private void TipPointerChange()
+    {
+        connector[0].Tooltip.ShowTipPointer = false;
+    }
+}
+```
+
+## Tooltip template content
+
+To customize the tooltip content or to create your own visualized element on the tooltip, [Template](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_Template) can be used.
```

Comment:
To customize the tooltip content or to create your own visualized element on the tooltip, the [Template](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_Template) can be used.